src/screens/PasswordRecovery/PasswordRecovery.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { Text, KeyboardAvoidingView, ScrollView } from "react-native";
+
+import { Button, Input } from "../../components";
+
+import { styles } from "./styles";
+
+const fields = {
+  email: "email"
+};
+
+const PasswordRecovery = props => {
+  const { emailDefault, hintText } = props;
+
+  const [{ email }] = useState({
```

Comment:
I guess there should be also setState destructuring, so that once value changed, field handler will update state value of the field and it will be possible to integrate service for sending email which require filled email value